draft-ietf-capport-api.md

```diff
@@ -61,17 +62,19 @@ The Captive Portal Architecture defines three steps of interaction between hosts
 2. API Server interaction, in which a host queries the state of the captive portal and retrieves the necessary information to get out of captivity
 3. Enforcement, in which the enforcement device in the network blocks disallowed traffic, and sends ICMP messages to let hosts know they are blocked by the captive portal
 
-This document is focused on the second step. It is assumed that the location of the Captive Portal API server has been provisioned as part of the first step. The mechanism for discovering this value not covered by this document.
+This document is focused on the second step. It is assumed that the location of the Captive Portal API server has been discovered by the host as part of the first step. The mechanism for discovering the API Server endpoint is not covered by this document.
 
 # API Details
 
-## URI of Captive Portal API
+## URI of Captive Portal API endpoint
 
-In order to communicate with the captive portal API server, the URI provisioned to the host SHOULD use the form of a well-known URI [RFC5785] ending with the string ".well-known/captive-portal.json". If this suffix is not included in the URI already provisioned, a host SHOULD append the suffix when communicating to the server.
+The URI of the API endpoint MUST be accessed using HTTP over TLS (HTTPS) and SHOULD be served on port 443 [RFC2818].
+The host SHOULD not assume that the URI will be the same each time. Depending on how the Captive Portal system is configured, the URI may be unique for each host and between session for the same host.
```

Comment:
When you say it won't be the "same each time", is "each time" referring to re-associating with the same network? It would be good to clarify. Perhaps:

```
The host SHOULD NOT assume that the URI for a given network attachment will stay the same, and should rely on the discovery or provisioning process each time it joins the network.
```

Reply:
thanks, i struggled with that one a bit and i like the language you proposed.